Google.PowerShell/Sql/GcSqlInstanceReplicaConfigCmdlet.cs

```diff
@@ -133,7 +163,8 @@ protected override void ProcessRecord()
                     Password = MySqlPassword,
                     SslCipher = MySqlSslCipher,
                     Username = MySqlUser,
-                    VerifyServerCertificate = MySqlVerifyCertificate
+                    VerifyServerCertificate = MySqlVerifyCertificate,
+                    Kind = "sql#mysqlReplicaConfiguration"
```

Comment:
Why is this necessary? Seems like an API wart. We shouldn't have to deal with this.